http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberSpec.scala

```diff
@@ -1781,6 +1781,60 @@ class LeoPubsubMessageSubscriberSpec
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
+  it should "handle Azure StartRuntimeMessage and start runtime" in isolatedDbTest {
+    val azurePubSubHandler = makeAzureInterp()
+    val leoSubscriber = makeLeoSubscriber(azureInterp = azurePubSubHandler)
+
+    val res = for {
+      disk <- makePersistentDisk().copy(status = DiskStatus.Ready).save()
+
+      azureRuntimeConfig = RuntimeConfig.AzureConfig(MachineTypeName(VirtualMachineSizeTypes.STANDARD_A1.toString),
+                                                     disk.id,
+                                                     azureRegion
+      )
+
+      runtime <- IO(
+        makeCluster(1)
+          .copy(status = RuntimeStatus.Starting, cloudContext = CloudContext.Azure(azureCloudContext))
+          .saveWithRuntimeConfig(azureRuntimeConfig)
+      )
+      tr <- traceId.ask[TraceId]
+      _ <- leoSubscriber.messageResponder(StartRuntimeMessage(runtime.id, Some(tr)))
+      updatedRuntime <- clusterQuery.getClusterById(runtime.id).transaction
+    } yield {
+      updatedRuntime shouldBe defined
+      updatedRuntime.get.status shouldBe RuntimeStatus.Starting
```

Comment:
I'm not sure if this validating anything...Runtime's status was initially `Starting` to begin with